### PR TITLE
add package.json to enable 'npm install'

### PR DESCRIPTION
**Pull request by Pavek**

### CODE REVIEW

**Comment by TooTallNate** [comment]
It appears that somebody has already published this code to npm: https://www.npmjs.com/package/filesaver.js

**Comment by wiebl** [comment]
https://github.com/mWater/FileSaver.js (source of the npm package) is outdated

**Comment by SpencerCarstens** [comment]
:+1:

File: package.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+    "name": "filesaver.js",
+    "version": "0.2.0",
```
Comment (eligrey) [inline package.json]: https://github.com/eligrey/FileSaver.js/blob/master/FileSaver.js#L3

**Comment by SpencerCarstens** [comment]
Should be 1.1.20160328 I think...

**Comment by SpencerCarstens** [comment]
What is the current status of this?

**Comment by graingert** [comment]
@eligrey ping
@DanlanNZ ping

Can we have one standard build for this.

**Comment by graingert** [comment]
@grassick ping

**Comment by DanlanNZ** [comment]
for some background there is [16 similar issues](https://github.com/eligrey/FileSaver.js/issues?utf8=%E2%9C%93&q=npm) related to this. correct me if I'm wrong but it looks like @eligrey's position on the matter is
> It's MIT licensed so you can easily upload any version to npm, as long as you are willing to maintain the npm metadata for your npm entry. I don't usually include support for specific package managers in my projects.
> 
> You can also just use git submodules to easily keep in sync me with me. I'm not stopping any of you from using npm. I just prefer to use GitHub instead.
https://github.com/eligrey/FileSaver.js/issues/177#issuecomment-159486757

I guess you could reference the package in your package.json using one of the following direct references but it's not ideal
```json
"dependencies": {
  "filesaverjs": "git://github.com/eligrey/FileSaver.js#commit-ish"
}
```
or
```json
"dependencies": {
  "filesaverjs": "https://github.com/eligrey/FileSaver.js/tarball/#commit-ish"
}
```

I'm happy to add contributors to the fork I've published to [npm](https://www.npmjs.com/package/filesaverjs) and help manage future releases of this

**Comment by elado** [comment]
Obviously, an NPM package is needed.
I currently use https://www.npmjs.com/package/browser-filesaver which seems to be up-to-date.

**Comment by jimmywarting** [comment]
been added

**Pull request closed without merging**